### ISSUE TITLE
Option to also sync "Ankündigungen", "E-Mails" and "Hyperlinks"

### ISSUE THREAD

**Issue by claell** [Sync-my-L2P/Sync-my-L2P#49]
An option to sync those folders as well (e.g. as .txt files; "Hyperlinks" maybe to one big .txt file containing all links) would be good.

**Comment by se-bi** [comment]
I would like to add here. that the solution of exercises should be synchronized as well.

**Comment by justin-time** [comment]
A possibility to show announcements e-mails or hyperlinks will maybe be add later. But this will be a bit more work than to support downloading assignment solutions.

The download of assignment solutions could be add in the next release.

**Comment by justin-time** [comment]
The API of the L2P does not support to download the sample solution. The API only supports to download the user created solution...

**Comment by claell** [comment]
Thanks for your good work @justin-time. Hope the other features will get added later then :) Also maybe one can talk to the creators of the API, so the allow the download of the sample solution as well (Since that is probably the solution mostly wanted).

**Comment by justin-time** [comment]
Sometimes is "later" = "very soon"!

There is a PR for supporting announcement and emails within L²P but without possibility to sync.

**Comment by claell** [comment]
Wow, that was fast, just saw the PR. Thank you very much. Will it be possible to sync as well in the future?

**Comment by justin-time** [comment]
No, sorry! There is no plan to add this feature because the messages use HTML and so there is no possibility to read this messages in a extern text editor in plain text without removing all HTML-tags.

**Comment by claell** [comment]
So will the plain html files get downloaded somewhere? It just would be good to have a way to store the messages since the rooms in the L2P are closed after some years.

**Comment by KaiDevelopment** [comment]
> The API of the L2P does not support to download the sample solution. The API only supports to download the user created solution...

Any news on this?
